lib/Doctrine/Task/BuildAll.php

```diff
@@ -35,30 +35,33 @@ class Doctrine_Task_BuildAll extends Doctrine_Task
     public $description          =   'Calls generate-models-from-yaml, create-db, and create-tables',
            $requiredArguments    =   array(),
            $optionalArguments    =   array();
-    
+
     protected $models,
               $tables;
-    
+
+    // This was undefined, added for static analysis and set to public so api isn't changed
+    public $createDb;
```

Comment:
Only used locally as far as I can tell, this is also true for the other Task classes that follow.